File: app/src/main/java/wxm/KeepAccount/ui/base/TouchUI/TouchPreviewColumnChartView.java
package wxm.KeepAccount.ui.base.TouchUI;

import android.annotation.SuppressLint;
import android.content.Context;
import android.util.AttributeSet;

import lecho.lib.hellocharts.view.PreviewColumnChartView;

/**
 * @author WangXM
 * @version create：2018/4/9
 */
public class TouchPreviewColumnChartView extends PreviewColumnChartView {
    public TouchPreviewColumnChartView(Context context) {
        super(context);
    }

    public TouchPreviewColumnChartView(Context context, AttributeSet attrs) {
        super(context, attrs);
    }

    public TouchPreviewColumnChartView(Context context, AttributeSet attrs, int defStyle) {
        super(context, attrs, defStyle);
    }

    @SuppressLint("ClickableViewAccessibility")
    @Override
    public boolean performClick() {
        // do what you want
        return true;
    }
}

File: app/src/main/java/wxm/KeepAccount/ui/base/TouchUI/TouchPreviewColumnChartView.kt
package wxm.KeepAccount.ui.base.TouchUI

import android.annotation.SuppressLint
import android.content.Context
import android.util.AttributeSet

import lecho.lib.hellocharts.view.PreviewColumnChartView

/**
 * @author WangXM
 * @version create：2018/4/9
 */
class TouchPreviewColumnChartView : PreviewColumnChartView {
    constructor(context: Context) : super(context) {}

    constructor(context: Context, attrs: AttributeSet) : super(context, attrs) {}

    constructor(context: Context, attrs: AttributeSet, defStyle: Int) : super(context, attrs, defStyle) {}

    @SuppressLint("ClickableViewAccessibility")
    override fun performClick(): Boolean {
        // do what you want
        return true
    }
}
